src/admin/paths/admin@expense_policies.yaml

```diff
@@ -0,0 +1,97 @@
+post:
+  tags:
+    - Expense Policies
+  summary: Create or Update Expense Policies
+  description: |
+    Create or Update Expense Policies <br>
+    > Apart from required fields, one of expense* or employee* and action* field must be set. <br>
```

Comment:
Apart from the required fields mentioned below, the request payload should contain the following:
- at least one expense_* or employee_* field
- at least one action_* field

Reply:
see, if somehow we can bring emphasis to this information